bot/answers.py

```diff
@@ -0,0 +1,187 @@
+'''
+File with all buttons and keyboards for bot
+'''
+
+# Matkup
+from telebot.types import ReplyKeyboardMarkup, KeyboardButton, \
```

Comment:
https://github.com/google/styleguide/blob/gh-pages/pyguide.md#32-line-length

Either don't break lines or break it adding parentheses, eg 

```py
from bla.bla import (
    MyVeryLongImport,
    LinesWhichDoNotFit,
    IntoOneLineRecommendedWidth
)
```